packages/ui/components/apps/appCard.test.tsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable playwright/missing-playwright-await */
```

Comment:
why do we need this?

Reply:
Without this is added automatically a "await" in some code lines, we already did this for another test PR ([example](https://github.com/calcom/cal.com/pull/10785/files#diff-1eafe4b11f56a6b7c180295a96bc8f4ed742c665598fefc5988d5f4b9d600daf))